topology/probes/ovs_of.go

```diff
@@ -265,6 +265,7 @@ var executor Execute = RealExecute{}
 func (r RealExecute) ExecCommand(com string, args ...string) ([]byte, error) {
 	/* #nosec */
 	command := exec.Command(com, args...)
+	defer command.Wait()
```

Comment:
CombinedOutput() call Run(), Run() call Wait()
https://golang.org/src/os/exec/exec.go?s=8356:8381#L275

Reply:
yep good catch, will remove it